docs/theory/diffusion/diffusion.rst

```diff
@@ -0,0 +1,713 @@
+    \langle\Delta x_n \Delta x_{n+1}\rangle & = & -\left(\sigma^2 - 2 R D \Delta t \right) &\\
+    \langle\Delta x_n \Delta x_m\rangle & = & 0 &\mathrm{, for }\left| n-m \right| > 1.
+    \end{align}
+
+Here :math:`D` represents the diffusion constant, :math:`sigma` is the localization uncertainty standard deviation and :math:`R` is the motion blur constant.
+In the current implementation, we assume the motion blur to be negligible for confocal scans (see note below).
+From these relations, one can derive the covariance based estimator :cite:`vestergaard2014optimal,vestergaard2015estimation`:
+
+.. math::
+
+    \begin{align}
+    \hat{D} =& \frac{\overline{\left(\Delta x_n\right)^2}}{2 \Delta t} + \frac{\overline{\Delta x_n \Delta x_{n+1}}}{\Delta t}\\
+    \hat{\sigma^2} =& R \overline{\left(\Delta x\right)^2} + (2 R - 1) \overline{\Delta x_n \Delta x_{n+1}}
+    \end{align}
+
+Here the bar indicates averaging over the time series.
+This method can be extended to handle tracks that have gaps due to blinking by only considering the successful localizations :cite:`vestergaard2016optimizing`.
+To take this into account, we replace :math:`\Delta t` with :math:`\overline{\Delta t_m}`, where :math:`t_m` indicates the timestep between the successful localizations :math:`m` and :math:`m+1` :cite:`vestergaard2016optimizing`.
+
+If the localization uncertainty is known beforehand, one can derive the following estimator:
+
+.. math::
+
+    \hat{D} = \frac{\overline{\left(\Delta x_n\right)^2} - 2 \sigma^2}{2 (1 - 2 R) \Delta t}
+
+.. note::
+
+    The motion blur coefficient :math:`R` is a value between `0` and `1/4` given by
+
+    .. math::
+
+        R = \frac{1}{\Delta t} \int_{0}^{\Delta t}S(t) \left(1 - S(t)\right)dt
+
+    with
+
+    .. math::
+
+        S(t) = \int_{0}^{t} c(t') dt'
+
+    Here, the aperture function is defined as :math:`c(t)`, where :math:`c(t)` represents the fraction of illumination happening before time `t`.
+    :math:`c(t)` is normalized such that :math:`S(0) = 0` and :math:`S(\Delta t) = 1`.
+    For a rectangular shutter or exposure window, one obtains :math:`R = \frac{1}{6} \frac{t_{exposure}}{line\_time}`.
+    In the current implementation we assume the motion blur constant to be negligible (zero) for confocal acquisition, since the time spent over the particle's PSF compared to the line time is very low.
+
+    When estimating both localization uncertainty and the diffusion constant, the motion blur factor has no effect on the estimate of the diffusion constant itself, but it does affect the calculated uncertainties.
+    In the case of a provided localization uncertainty, it does impact the estimate of the diffusion constant itself.
+
+The performance of these covariance-based estimators can be characterized by its signal to noise ratio (SNR).
+This SNR is defined by:
+
+.. math::
+
+    \mathrm{SNR} = \frac{\sqrt{D \Delta t}}{\sigma}
```

Comment:
Maybe the Pylake docs is not the place for this. But it would be great to have a guideline somewhere of what $\sigma$ is for typical experiments. And is $\Delta t$ a self-chosen time step? That way users can compute the SNR for their own experiments.

Reply:
$\Delta t$ is the kymograph line time. Estimates for $\sigma$ are returned from the CVE estimator as a result.

I agree it's a good thing to include; but I don't feel confident providing those numbers in the docs yet at this point.

Reply:
Yeah, $\sigma$ is a kind of spec, that does not belong in the Pylake docs. It is indeed best to just give instructions on how to estimate this quantity, rather than giving the value.